pandas/core/algorithms.py

```diff
@@ -910,7 +910,7 @@ def duplicated(
 
     Parameters
     ----------
-    values : ndarray-like
+    values : nd.array, ExtensionArray or Series
```

Comment:
this doesn't now match the type annotation.

Reply:
Created https://github.com/pandas-dev/pandas/issues/42604